app/realtime.php

```diff
@@ -90,6 +125,8 @@ function () use ($register, $db, $cache) {
             Authorization::enable();
             $documentId = $document->getId();
         } catch (\Throwable $th) {
+            call_user_func($logError, $th, "createWorkerDocument");
+
             Console::error('[Error] Type: ' . get_class($th));
```

Comment:
Can we move all the Console::error logs to the error callback?

Reply:
We can. Moved it.